frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -216,6 +216,14 @@
           Aucune cantine n'a renseigné des données relatives à la loi EGalim pour l'année {{ year }}.
         </p>
       </div>
+      <div v-else-if="year === 2024">
+        <DsfrCallout class="my-6">
+          <p>
```

Comment:
```suggestion
          <p class="mb-0">
```
On peut supprimer le margin-bottom par défaut du DSFR comme il n'y a pas d'autre paragraphe en dessous